scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -1456,10 +1458,20 @@
                                    (when-let [job (tools/job-ent->map (:job/_instance task-ent))]
                                      (let [pool-name (cached-queries/job->pool-name job)
                                            task-request (make-task-request db job pool-name :task-id task-id)
-                                           ^TaskScheduler fenzo (pool->fenzo pool-name)]
+                                           {:keys [^TaskScheduler fenzo unassign-task-set]} (pool-name->fenzo-state pool-name)
+                                           to-unassign (util/set-atom! unassign-task-set #{})
```

Comment:
Why not use [`reset-vals!`](https://clojuredocs.org/clojure.core/reset-vals!) instead of `util/set-atom!`?

Reply:
@scrosby Thoughts on this?